package.json

```diff
@@ -14,6 +14,8 @@
     "lint:fix": "lerna run lint:fix",
     "start": "npm start --prefix packages/template-retail-react-app",
     "pretest": "npm run lint",
+    "test:e2e": "npx playwright test",
```

Comment:
We should update the e2e workflow as well to keep things consistent.
Replace [npx plawright test](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/bfd592c9644297a897b8c031837ff6819c5b7377/.github/workflows/e2e.yml#L120) with the npm command.

NOTE: There are multiple instances in the same file.